lib/service/backup_service.dart

```diff
@@ -114,27 +122,30 @@ class BackupService {
     log.info('[BackupService][start] deleteAllProfiles');
     String? deviceId = await getBackupId();
     final endpoint = Environment.autonomyAuthURL;
-    final authToken =
-        await injector<AuthService>().getAuthTokenByAccount(account);
+    final authToken = await injector<AuthService>().getAuthToken();
 
     await http.delete(Uri.parse('$endpoint/apis/v1/premium/profile-data'),
-        headers: {'requester': deviceId, 'Authorization': 'Bearer $authToken'});
+        headers: {
+          'requester': deviceId,
+          'Authorization': 'Bearer ${authToken?.jwtToken}'
+        });
   }
 
-  Future restoreCloudDatabase(WalletStorage account, String version,
-      {String dbName = 'cloud_database.db'}) async {
+  Future restoreCloudDatabase({String dbName = 'cloud_database.db'}) async {
```

Comment:
if you always check version before restore db, please pass that version as param, don't call it again